File: beginner/java/2708.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        String action;
        int n, cont1 = 0, cont2 = 0;

        while(true){

            action = scanner.next();

            if(action.equals("ABEND"))
                break;
            else{
                n = scanner.nextInt();

                if(action.equals("SALIDA")){
                    cont1+=n;
                    cont2+=1;
                } else{
                    cont1-=n;
                    cont2-=1;
                }
            }
        }

        System.out.printf("%d%n%d%n", cont1, cont2);
    }
}

File: beginner/c/2708.c
#include <stdio.h>
#include <string.h>

int main() {
    char action[50];
    int i, n, cont1 = 0, cont2 = 0;

    while(1){

        scanf("%s", action);

        if(strcmp(action, "ABEND") == 0)
            break;
        else{
            scanf("%d", &n);
            
            if(strcmp(action, "SALIDA") == 0){
                cont1+=n;
                cont2+=1;
            } else{
                cont1-=n;
                cont2-=1;
            }
        }
    }

    printf("%d\n%d\n", cont1, cont2);

    return 0;
}
